code/modules/clothing/sensors/_sensor.dm

```diff
@@ -0,0 +1,12 @@
+/obj/item/clothing/sensor
+	name = "sensor"
+	abstract_type = /obj/item/clothing/sensor
+	slot_flags = SLOT_TIE
+	icon_state = ICON_STATE_WORLD
+	accessory_slot = ACCESSORY_SLOT_SENSORS
```

Comment:
does this mean you won't be able to have both a buddy tag and a vitals sensor on the same uniform?